src/script.js

```diff
@@ -165,14 +162,14 @@ function composeReplyTable(response,columns,data) {
             if(item[key]){
                 flag=true;
                 var tdItem = document.createElement("td");
-                if(key==="web_pages") {
```

Comment:
@0x48piraj why are you removing these lines?

Reply:
I haven't  :open_mouth:

Reply:
fixed

Reply:
@zamhaq Fixed it!